src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -112,6 +123,14 @@ class ChaptersButton extends TextTrackButton {
     if (chaptersTrack && chaptersTrack.cues && chaptersTrack.cues.length > 0) {
       let cues = chaptersTrack['cues'], cue;
 
+      // We want to empty out the item children but just keep the chapter title item
+      if (menu.children_.length > 2) {
```

Comment:
we should try and avoid accessing "private" variables if there are other ways of removing things. Given that each item is added as a child, we should instead remove each child using the appropriate API method.